homeassistant/components/reolink/update.py

```diff
@@ -41,7 +41,7 @@ class ReolinkUpdateEntity(
 
     _attr_device_class = UpdateDeviceClass.FIRMWARE
     _attr_release_url = "https://reolink.com/download-center/"
-    _attr_name = "Update"
+    _attr_translation_key = "update"
```

Comment:
Update what? Currently it's `update.probably_some_camera_update` but what are you updating? Is it firmware?

Reply:
Yes it is the firmware of the device (with the device name prepended) so this is or the firmware of the NVR or the firmware of the camera.
Note it has the device_class UpdateDeviceClass.FIRMWARE

Schould I change the name to something else?

Reply:
Oh now that I look on my pc I see the device class. In #98579 device class translations will also be added for Update entities. So if you wait for that PR (I don't think that PR will take long before being merged) you can omit the translation key and it would directly turn into `update.probably_some_camera_firmware`.

Reply:
Yes, I will wait for that PR, already made the changes in this PR (but now the name is missing so the checks fail, so need the merge).

Reply:
The PR is merged!

Reply:
Thanks for letting me know, I have merged the changes to this PR.